apps/api_web/lib/api_web/router.ex

```diff
@@ -28,11 +28,12 @@ defmodule ApiWeb.Router do
 
     @content_security_policy Enum.join(
                                [
-                                 "default-src 'none'",
+                                 "default-src 'self'",
```

Comment:
question: I had left this out because not all of the options needed `'self'` (namely fonts). Is there a reason to add it back?

Reply:
> question: I had left this out because not all of the options needed `'self'` (namely fonts). Is there a reason to add it back?

Reverted this, it does still work without it. I had changed this prior to the other policies 🙂